compiler/test/__snapshots__/enums.aa34084a.0.snapshot

```diff
@@ -118,7 +118,7 @@ enums › adt_trailing
           )
           (i64.store offset=24
            (local.get $0)
-           (i64.const 85899347075)
+           (i64.const 85899347051)
```

Comment:
This code populates the type information in memory that's used for printing. The stamp is what is used as the identifier for enums at runtime, so this changes to reflect that.

Reply:
Thanks 🙏  This was going to be my only question